### PR TITLE
option added to enable anyuid scc  - defaults to off

### PR DESCRIPTION
**Pull request by tchughesiv**
Adding `scc_anyuid: True` to the my_vars.yml file... will enable anyuid for authenticated users.

addresses this issue:
https://github.com/fusor/catasb/issues/91

### CODE REVIEW

File: local/linux/README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ These playbooks will do the following in a local environment:
       ```
   * Install python dependencies
     ```bash
-    $ pip install six
+    $ pip install six==1.10.0 docker==2.4.2 ansible==2.3.0
```
Comment (tchughesiv) [inline local/linux/README.md]: starting today, i was experiencing issues when issuing `pip install -U six docker ansible`.  Specifying these versions w/o a -U did the trick for me.  Let me know if you'd like to keep this commit or not.

**Comment by tchughesiv** [comment]
my only Q is... do we wait for this feature to be offered before locking down default catasb cluster?
https://trello.com/c/enjZJ6sp/299-3-document-design-for-locking-down-service-accounts-for-launching-apbs

I think, we probably move forward w/ merging this change w/o that feature because we want image devs to start thinking about deploying in a restricted scc... and, honestly, anyuid doesn't account for images that require privilege anyway.  So, that scc isn't an all encompassing role... we should encourage the most secure/default scc mode.

File: local/linux/README.md (hunk 1 above)
Comment (jwmatthews) [inline local/linux/README.md]: +1 thx for the note

**Comment by jwmatthews** [comment]
+1 to merging now.

I tested this PR on local/linux, saw success.

**Pull request merged**